core/src/main/java/org/radargun/stages/monitor/MonitorStartStage.java

```diff
@@ -6,18 +6,11 @@
 import org.radargun.reporting.Timeline;
 import org.radargun.stages.AbstractDistStage;
 import org.radargun.state.MasterState;
-import org.radargun.sysmonitor.AbstractMonitors;
-import org.radargun.sysmonitor.CpuUsageMonitor;
-import org.radargun.sysmonitor.GcMonitor;
-import org.radargun.sysmonitor.InternalsMonitor;
-import org.radargun.sysmonitor.MasterMonitors;
-import org.radargun.sysmonitor.MemoryUsageMonitor;
-import org.radargun.sysmonitor.NetworkBytesMonitor;
-import org.radargun.sysmonitor.OpenFilesMonitor;
-import org.radargun.sysmonitor.SlaveMonitors;
+import org.radargun.sysmonitor.*;
```

Comment:
Could you configure your IDE to not use wildcard imports, please?

Reply:
Should we update the IDE settings? Today it is set equals 10.
https://github.com/radargun/radargun/blob/3359ca7f6174116b1844719685fe8ee7a5123c85/ide-settings/intellij/RadarGunCodeStyle.xml#L17

Reply:
Hmm, I didn't know about these settings. Yes, no wildcards is better.

Reply:
https://github.com/radargun/radargun/pull/541/files